app/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Document</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <div class="home-page">
+      <header class="home-page__header">
```

Comment:
If you want the logo to be in the `<header>`, then your solution is good.
In general, you can take the logo off the header.
homepage will have as children:
logo
header
menu
content
footer
You will position all of these children through .home-page grid system.
In this case you will not have to set property `display: grid;` to header => less code for the same layout